libtransmission/quark.cc

```diff
@@ -137,6 +137,7 @@ auto constexpr MyStatic = std::array<std::string_view, TR_N_KEYS>{
     "fromLtep"sv,
     "fromPex"sv,
     "fromTracker"sv,
+    "gateway-address"sv,
```

Comment:
_(nit)_ Use `gateway_address` as the string literal.

The existing strings' style is an inconsistent mess, but new strings should all be camel_case